test/helpers/app.js

```diff
@@ -10,8 +9,11 @@ const truncateTable = model =>
 
 exports.truncateDatabase = () => Promise.all(tables.map(truncateTable));
 
-exports.getResponse = ({ endpoint, header = {}, body, method = 'put' }) =>
-  request(app)
+exports.getResponse = ({ endpoint, header = {}, body, method = 'put' }) => {
+  // eslint-disable-next-line global-require
+  const app = require('../../app.js');
```

Comment:
Why this change?

Reply:
Because all test are independent. Before this change, we used it the same app for all test, but I think that it's wrong. Each test must get response for any endpoint with your app.